src/foam/u2/crunch/wizardflow/StepWizardAgent.js

```diff
@@ -37,34 +38,17 @@ foam.CLASS({
       value: {
         class: 'foam.u2.wizard.StepWizardletView',
       }
-    },
-    {
-      name: 'overrideTopCapabilityConfig',
-      documentation: `
-        Set this to true and this StepWizardAgent's config will override
-        any wizard configuration specified by the root capability being
-        granted.
-      `
     }
   ],
 
   methods: [
     function execute() {
-      let topCap = this.capabilities.slice(-1)[0];
-      let topCapConfig = topCap.wizardletConfig.cls_.create({
-        ...topCap.wizardletConfig.instance_
-      }, this);
-      let config = this.overrideTopCapabilityConfig
-        ? topCapConfig.copyFrom(this.config)
-        : this.config.copyFrom(topCapConfig)
-        ;
```

Comment:
This shouldn't be removed, but since it's currently broken I'll re-implement a solution after this is merged.